docs/en/guide/devtool/trace/record-trace.mdx

```diff
@@ -10,27 +10,29 @@ import * as NextSteps from '@lynx/NextSteps';
 
 # Record Trace
 
-This article provides a comprehensive guide on how to use the Devtool to record performance data for Lynx pages. By following these steps, you will be able to capture detailed information about the page rendering process and engine execution, enabling effective performance analysis and problem diagnosis.
+This article aims to help you use DevTool to record Trace for Lynx pages. By following these steps, you will be able to capture detailed information about the page rendering process and engine execution, enabling effective performance analysis and problem diagnosis.
 
 ## Setup
 
-1. Install [Lynx Devtool Desktop Application](/guide/devtool/panels.html#run-lynx-devtool-desktop-application);
-2. Use the latest [LynxExplorer app](https://github.com/lynx-family/lynx/releases) or [integrate lynx dev version](/guide/start/integrate-lynx-dev-version.html) into your application;
-3. Use a data cable to connect your device to your computer;
+1. Install [Lynx DevTool Desktop Application](/guide/devtool/panels.html#run-lynx-devtool-desktop-application).
```

Comment:
Lynx DevTool -> DevTool.

All the website is talking Lynx. No need for lynx prefix